io.openems.edge.evcs.api/src/io/openems/edge/evcs/v2/api/SocEvcs.java

```diff
@@ -0,0 +1,98 @@
+package io.openems.edge.evcs.v2.api;
+
+import io.openems.common.channel.AccessMode;
+import io.openems.common.channel.PersistencePriority;
+import io.openems.common.channel.Unit;
+import io.openems.common.types.OpenemsType;
+import io.openems.edge.common.channel.Doc;
+import io.openems.edge.common.channel.IntegerReadChannel;
+import io.openems.edge.common.channel.value.Value;
+import io.openems.edge.common.modbusslave.ModbusSlaveNatureTable;
+import io.openems.edge.common.modbusslave.ModbusType;
+
+public interface SocEvcs extends Evcs {
+
+	public enum ChannelId implements io.openems.edge.common.channel.ChannelId {
+
+		/**
+		 * Current SoC.
+		 *
+		 * <p>
+		 * The current state of charge of the car
+		 *
+		 * <ul>
+		 * <li>Interface: SocEvcs
+		 * <li>Readable
+		 * <li>Type: Integer
+		 * <li>Unit: Percent
+		 * </ul>
+		 */
+		SOC(Doc.of(OpenemsType.INTEGER) //
+				.unit(Unit.PERCENT) //
+				.accessMode(AccessMode.READ_ONLY) //
+				.persistencePriority(PersistencePriority.HIGH)); //
+
```

Comment:
Is it possible to add something like REMAINING_ENERGY with the unit Wh as a Channel? Just the percentage seems unlikely to be useful.
This channel would indicate how much energy is needed to charge the Battery to 100% and could be used for optimizing the charge duration.
Also a Channel for CURRENT_CAPACITY in Wh would be useful to put in the frontend. A user could therefor do a remaining distance calculation.
As an alternative we could use MAXIMUM_CAPACITY for a basis of calculation like in SymmetricESS.

Reply:
we should use similar namings to the iso, than it is often easier to understand
https://www.typhoon-hil.com/documentation/typhoon-hil-software-manual/References/iso15118_protocol.html#iso15118_protocol__section_e5d_thl_ctb
energy_capacity

and than you can calculate the rest out of it. 
evtl call it ISO15118EVCS?

Reply:
@TimStoecker, @wimmerp, both good points. Also I like the idea of naming it appropriate to ISO15118. I personally would prefer V2gEvcs or somethink like that. Right now I am not aware of a chargepoint providing this information. 
I would leave the interface like it is for now.
AFAIK we will have a bidirectional chargepoint on the hackaton.  We could decide it there.